shared/templates/audit_rules_privileged_commands/kubernetes.template

```diff
@@ -13,7 +13,7 @@ spec:
     storage:
       files:
       - contents:
-          source: data:,-a%20always%2Cexit%20-F%20path%3D{{{ PATH }}}%20-F%20auid%3E%3D{{{ auid }}}%20-F%20auid%21%3Dunset%20-F%20key%3Dprivileged%0A
+          source: data:,-a%20always%2Cexit%20-F%20arch%3Db32%20-F%20path%3D{{{ PATH }}}%20-F%20auid%3E%3D{{{ auid }}}%20-F%20auid%21%3Dunset%20-F%20key%3Dprivileged%0A-a%20always%2Cexit%20-F%20arch%3Db64%20-F%20path%3D{{{ PATH }}}%20-F%20auid%3E%3D{{{ auid }}}%20-F%20auid%21%3Dunset%20-F%20key%3Dprivileged%0A
```

Comment:
This doesn't seem right. 

The part that concerns me the most`-F arch`

Reply:
I think you mean the missing perm_x there. I have add it now. But, it won't be used because the rhcos4 product doesn't set this perm_x.